apps/nextjs/src/app/[locale]/manage/users/[userId]/_components/profile.accordion.tsx

```diff
@@ -50,7 +68,11 @@ export const ProfileAccordion = ({ user }: ProfileAccordionProps) => {
           label={t("user.field.email.label")}
           {...form.getInputProps("email")}
         />
-        <Button type="submit" disabled={!form.isValid()} loading={isPending}>
+        <Button
+          type="submit"
+          disabled={!form.isValid() || !form.isDirty()}
```

Comment:
Here to disallow submitting an edit request when the form is not touched

Reply:
You may have to run `setInitialValues` on the form with the new values after a successful update so it's no longer dirty and has the correct initial values